helm-charts/FATE/values.yaml

```diff
@@ -189,6 +189,7 @@ modules:
     storageClass:
     accessMode: ReadWriteOnce
     size: 1Gi
+    notebook_hashed_password: 
```

Comment:
Also, please remember to change the config doc.

https://github.com/FederatedAI/KubeFATE/blob/3d88f3305ffd769cd3ab8bc21ca4d374b153b93d/docs/configurations/FATE_cluster_configuration.md